src/scene.ts

```diff
@@ -2546,16 +2548,23 @@ export class Scene extends AbstractScene implements IAnimatable {
      * @param projection defines the Projection matrix to use
      */
     public setTransformMatrix(view: Matrix, projection: Matrix): void {
-        if (this._viewUpdateFlag === view.updateFlag && this._projectionUpdateFlag === projection.updateFlag) {
+        this._setMultiviewTransformMatrix(view, projection);

```

Comment:
why not adding the additional parameter to setTransformmatrix? The _setMultiviewTransformmatrix is a bit misleading as it also takes care of non multiview version